tests/components/yale/fixtures/get_doorbell.json

```diff
@@ -0,0 +1,81 @@
+{
+  "status_timestamp": 1512811834532,
+  "appID": "august-iphone",
```

Comment:
There is still some use of "august" in the test fixtures.

Reply:
This is expected as the yale (company) code was forked from the original august code and they still have august all over the place internally.

Reply:
August was sole to [ASSA ABLOY](https://www.assaabloy.com/) than it was spun off into yet another company "Yale August" aka `august`.  

The `yale` integration will be for what the "ASSA ABLOY" company is doing.   
The `august` integration will be for what the "Yale August" company is doing.

The original August company devices are folding into `Yale August`